runtime/calamari/src/lib.rs

```diff
@@ -755,7 +757,7 @@ impl cumulus_pallet_dmp_queue::Config for Runtime {
 
 parameter_types! {
 	// Rotate collator's spot each 6 hours.
-	pub const Period: u32 = 6 * HOURS;
+	pub Period: u32 = prod_or_fast!(6 * HOURS, 2 * MINUTES,"CALAMARI_PERIOD");
```

Comment:
There's one more period in `pallet_treasury` that can be reduced to like 2 minutes.

Reply:
Thanks, added in https://github.com/Manta-Network/Manta/pull/383/files#diff-56417537fb560c5d1d8018408551510b0434860ead99a3ff28512ebbb4f669ddR529